packages/astro/src/core/create-vite.ts

```diff
@@ -131,7 +142,6 @@ export async function createVite(
 			astroScriptsPageSSRPlugin({ settings }),
 			astroHeadPlugin(),
 			astroScannerPlugin({ settings, logger }),
-			astroInjectEnvTsPlugin({ settings, logger, fs }),
```

Comment:
Not intended, I assume

Reply:
Hmm, took a deeper look to refresh my memory and no longer see any code running this besides `sync`. Reverting this!